mvcc/key_index.go

```diff
@@ -23,9 +23,7 @@ import (
 	"go.uber.org/zap"
 )
 
-var (
-	ErrRevisionNotFound = errors.New("mvcc: revision not found")
-)
+var ErrRevisionNotFound = errors.New("mvcc: revision not found")
```

Comment:
Not really important, but if other errors are added in the future, leaving this as a variable block would make it easier to track changes.